<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccessibleResourcesWidget</class>
 <widget class="QWidget" name="AccessibleResourcesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>331</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QnResourcePreviewWidget" name="previewWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nameLabel"/>
     </item>
     <item>
      <widget class="QLabel" name="descriptionLabel">
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="verticalLine">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QnTreeView" name="resourcesTreeView">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="autoExpandDelay">
      <number>500</number>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnResourcePreviewWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/resource_preview_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">ui/widgets/common/tree_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
